### ISSUE TITLE
About Prepocess

### ISSUE THREAD

**Issue by igo312** [kratzert/finetune_alexnet_with_tensorflow#21]
Hello there
In the datagenerator I find the VGG ,can you tell we what it is .
And I find that you use it to make the data centerned 
But for the variance is required to equal to one , how to solve it 
I really do not know how to do the preprocess , may I was wrong 
Hope for response , thank you!

**Comment by kratzert** [comment]
Oh this is just a bad naming from my side..I think it's because I copied this line from another code of mine. So VGG_MEAN is the the ImageNet mean. It's the per channel mean of all images in the ImageNet dataset, that must be subtracted from each Input image to centralize you samples. I will rename the variable to be more clear here. 
Thanks for the hint!

**Comment by igo312** [comment]
Well,thanks for answering.
May I ask one more question ? Cause I really do it by myself.
So I don't know whether the network works. My computer is not really good
I give 5 images to  the network  every time .Possibly after forty times ,I cannot see any progress 
So it means my network doesn't work , right?

**Comment by kratzert** [comment]
Hey igo312,
if you learn all this by yourself, you might take one of the wonderful online courses that exist and learn some of the basics of working with conv nets before trying to apply them. You might find the fastai video series good, or the cs231n or anything similar. Also my [blog post](https://kratzert.github.io/2017/02/24/finetuning-alexnet-with-tensorflow.html) to this article answers your question: You usually need a lot of images to train any of this networks, like 100s or 1000s for fine tuning and millions for training from scratch. So yes, 5 images might be the problem.

**Comment by igo312** [comment]
Wow, thanks for answering,
I just think you may get a lot of experience about this to judge if my networks work though with a low accuracy which has been feed about two hundred images already. 
Well , thanks for advice,  and also for the attitude.I know my English is not really good. Thanks a lot
